term.go

```diff
@@ -285,13 +285,10 @@ func (t *Terminal) run() {
 			fyne.LogError("pty read error", err)
 		}
 
-		lenLeftOver := len(leftOver)
-		if lenLeftOver > 0 {
-			buf = append(leftOver, buf[:num]...)
```

Comment:
This was appending to something that could run out space!  Leftover overfilled buf which it was doing.